pkg/audit/manager.go

```diff
@@ -433,11 +442,39 @@ func (am *Manager) auditResources(
 	return nil
 }
 
+func (am *Manager) auditFromCache(ctx context.Context) ([]Result, error) {
+	objs, err := am.auditCache.ListObjects(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("unable to list objects from audit cache: %w", err)
+	}
+
+	var results []Result
+
+	for i := range objs {
+		// Prevent referencing loop variables directly.
+		obj := objs[i]
+		resp, err := am.opa.Review(ctx, obj)
+		if err != nil {
+			return nil, err
```

Comment:
Should err here skip the error and continue with the rest of the loop instead? I think that is closer to the current behavior.

Reply:
Good call. IMO either log-and-swallow the errors, or returning results + some aggregated error object would prevent an error evaluating one constraint from short-circuiting the audit process.

Reply:
Ah, yeah. Sounds like this brings the "from cache" behavior to parity with the "from scraped" behavior.